docs/source/release_notes/release-notes.rst

```diff
@@ -35,41 +35,64 @@ Changelog
 develop
 =======
 
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+0.27.0
+=======
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
 
     *    - Type
          - Change
 
+    *    - |new|
```

Comment:
Is there any reason a user _wouldn't_ want to use stream store compression? I think we should be more emphatic that this exists and should be used in preference to non-compressed streams. Any reason this isn't the default?